docs/contents.rst

```diff
@@ -25,7 +25,7 @@
    
    data-collect-prep/ingest-process-data
    model_development/continuous-model-development
-   serving/serving-overview
+   deployment/deploy-models-and-ml-applications
```

Comment:
i prefer to use short names (e.g. `deployment`) a. shorter URLs, b. the way we call (full name) each section may change, so no need to make it too long and exact

Reply:
The problem is that we can end up with more than one file with the same name. We need enough to make each one unique (and somewhat self explanatory)

Reply:
this is a `main` category, its ok to make it short